### PR TITLE
Document testing that occurs during PR

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,4 +28,47 @@ You must have [cekit](https://github.com/cekit/cekit) installed.
...
+The features test execution has been split into multiple executions due to created images resource consumption and log size.
+Each feature file has its own execution with its own log.
+When a failure occurs:
+* Check the Github action execution log. Note the feature file name that failed.
+* Download the log files.
+* Unzip the log files and search for the `wildfly-s2i-test-logs-ubuntu-latest-<jdk>/**/test-logs-legacy-s2i-ubuntu-latest-jdk11.txt<feature name>-ubuntu-latest-<jdk>.txt` file.
+* Access the end of the file: `tail -f --lines 100 <path of the file>`
+* You should see something like:
+```
+Failing scenarios:
```
Comment (mchoma) [inline README.md:60]: Can presence of failing scenarios be indicated in Github UI, so one know he should dig deeper what is going on?

Comment (jfdenise) [reply README.md]: Good point. I added it. BTW, I added an extra transient commit to make the first executed test to fail.

File: README.md [hunk 2]
```diff
@@ -28,4 +28,47 @@ You must have [cekit](https://github.com/cekit/cekit) installed.
...
+  features/image/no-jdk11-legacy-s2i.feature:4  Test preview FP and preview cloud FP with legacy app.
+
+1 feature passed, 1 failed, 0 skipped
+9 scenarios passed, 1 failed, 0 skipped
+38 steps passed, 1 failed, 3 skipped, 0 undefined
+Took 8m17.543s
+```
+You can then grep the feature name inside the file, for example (`Test preview FP and preview cloud FP with legacy app.`) and look at the failure.
+
+NOTE: The S2I build logs contain ERROR traces that are not actual errors, just traces. For example:
```
[the remark this answers is not in the document]
Comment (jfdenise) [reply README.md:59]: We can remove 99% of the traces by filtering out the traces created when tar process occurs. All other traces can be of interest. I am going to implement this filtering.

**Comment by jfdenise** [comment]
@mchoma we have this run that shows the failing scenario and zipped files contain logs cleared from noise.
https://github.com/wildfly/wildfly-s2i/actions/runs/9288572407/job/25560354971?pr=476

**Pull request merged**